frontend/integration-tests/tests/crud.scenario.ts

```diff
@@ -332,7 +336,9 @@ describe('Kubernetes resource CRUD operations', () => {
     const labelValue = 'appblah';
 
     beforeAll(async () => {
-      await browser.get(`${appHost}/k8s/ns/${testName}/${plural}/~new`);
+      await browser.get(`${appHost}/k8s/ns/${testName}/${plural}`);
+      await crudView.isLoaded();
+      await crudView.createYAMLButton.click();
```

Comment:
Why was this change needed?

Reply:
I tested the pink and green code, back and forth, numerous times, and am convinced that FF needs the green code.

Reply:
> I tested the pink and green code, back and forth, numerous times, and am convinced that FF needs the green code.

We should understand *why*, though. (I'm worried there is a Firefox-specific bug in console here since this should work.)

Reply:
Is there a specific resource this fails on consistently?